dynatrace-operator-google-marketplace/schema.yaml

```diff
@@ -122,19 +413,48 @@ properties:
                   - get
                   - list
                   - watch
+              - apiGroups:
+                  - ""
+                resources:
+                  - nodes
+                  - namespaces
+                verbs:
+                  - get
+                  - list
+                  - watch
+              - apiGroups:
+                  - ""
+                resources:
+                  - secrets
+                verbs:
+                  - create
+              - apiGroups:
+                  - ""
+                resources:
+                  - secrets
+                resourceNames:
+                  - dynatrace-dynakube-config
+                  - dynatrace-dynakube-pull-secret
+                verbs:
+                  - get
+                  - update
+                  - delete
           - type: Role
             rulesType: CUSTOM
             rules:
               - apiGroups:
                   - dynatrace.com
                 resources:
                   - dynakubes
+                  - oneagentapms
```

Comment:
we don't need to have oneagent and oneagentapm here

Reply:
removed